changelogs/fragments/user_modules_type_annotation.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Added type annotation to user modules. (https://github.com/ScaleComputing/HyperCoreAnsibleCollection/pull/96)
```

Comment:
This is probably not needed.
Document fragments should be used in case of an "operational/functional" change.
Examples: Bugfix, adding/removing/altering input or output, new module, new functionality.

Reply:
I agree, I removed the fragment